go-selinux/selinux_linux.go

```diff
@@ -389,7 +389,7 @@ func writeCon(fpath, val string) error {
 		_, err = out.Write(nil)
 	}
 	if err != nil {
-		return &os.PathError{Op: "write", Path: fpath, Err: err}
+		return err
 	}
 	return nil
```

Comment:
Surprised the linter didn't complain that this could just `return err` (which could be nil, or not).

I'm ok with the current version as the linter is ok with it (and it may be somewhat clearer)